public/.well-known/assetlinks.json

```diff
@@ -0,0 +1,9 @@
+[{
+  "relation": ["delegate_permission/common.handle_all_urls"],
+  "target": {
+    "namespace": "android_app",
+    "package_name": "org.gooddollar",
+    "sha256_cert_fingerprints":
+    ["FA:C6:17:45:DC:09:03:78:6F:B9:ED:E6:2A:96:2B:39:9F:73:48:F0:BB:6F:89:9B:83:32:66:75:91:03:3B:9C"]
```

Comment:
@MarianoAmado @mauriguz 
the appIds and sha should probably include both dev + prod apps

Reply:
@MarianoAmado @mauriguz for ios its the same app id for dev and prod?
once you make sure, this can be merged

Reply:
@sirpy 
Yes, it's the same, both the bundle id (org.gooddollar) & the team (97K3Q6WTLM)